common-types/index.d.ts

```diff
@@ -191,3 +191,13 @@ interface Shoutout {
   readonly hidden: boolean;
   readonly uuid: string;
 }
+
+interface CoffeeChat {
+  readonly uuid: string;
```

Comment:
As you can see (from above), though we never followed this rule, we theoretically should be making all these fields `readonly`. Don't worry about the above ones but let's fix this one.